tile_caching/Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM mendlik/nginx:1.0.0
+FROM nginx:mainline-alpine
```

Comment:
The version is not fixed!

Reply:
I think it is - mainline-alpine is the tag, see https://hub.docker.com/_/nginx/tags

Reply:
Indeed `mainline-alpine` is a tag, but it's not fixed to a specific nginx version. Once a new mainline alpine or nginx version gets released the tag point's to the latest version. That is not what we want.

We should use `1.23.2-alpine` instead.